packages/app/src/components/FlameColorEditor/FlameColorEditor.tsx

```diff
@@ -13,14 +13,15 @@ import { createZoom, WheelZoomCamera2D } from '@/lib/WheelZoomCamera2D'
 import { createAnimationFrame } from '@/utils/createAnimationFrame'
 import { createDragHandler } from '@/utils/createDragHandler'
 import { eventToClip } from '@/utils/eventToClip'
-import { maxLength2 } from '@/utils/maxLength'
 import { scrollIntoViewAndFocusOnChange } from '@/utils/scrollIntoViewOnChange'
 import { wgsl } from '@/utils/wgsl'
 import ui from './FlameColorEditor.module.css'
 import type { v2f } from 'typegpu/data'
 import type { FlameFunction } from '@/flame/flameFunction'
 import type { HistorySetter } from '@/utils/createStoreHistory'
 
+const LIGHTNESS = 0.68
```

Comment:
Is this also adjusted for some image, or something that perhaps user should want to modify, or is it generic and works the best in all cases?

Reply:
this is just related to color editor. The points don't really have lightness. It is currently decided based on histogram counts.